src/Components/Movies.js

```diff
@@ -0,0 +1,71 @@
+import React, { Component } from "react";
+import "../Styles/Movies.css";
+import axios from "axios";
+
+export default class Movies extends Component {
+  state = {
+    movies: [],
+    selectedmovie: "",
+    title: "",
+    releaseDate: "",
+    description: "",
+  };
+
+  loadMovies = async () => {
+    try {
+      const res = await axios.get("https://ghibliapi.herokuapp.com/films");
+      const movieData = res.data;
+      this.setState({ movies: movieData });
+    } catch (err) {
+      console.log(err);
+      this.setState({ movies: [] });
+    }
+  };
+
+  componentDidMount() {
+    this.loadMovies();
+  }
+
+  handleChange = async (e) => {
+    this.setState({ selectedMovie: e.target.value });
+    try {
+      const res = await axios.get(
+        `https://ghibliapi.herokuapp.com/films/${e.target.value}`
+      );
+      const title = res.data.title;
```

Comment:
I think it's better to have one piece of state that keeps track of the entire movie and you can access the title and other pieces from that